kernel/test/KernelProxy.test.js

```diff
@@ -38,9 +38,9 @@ contract('zeppelin_os', ([_, zeppelin, developer, someone, anotherone]) => {
 
     // deploy another instance of the testing contract
     const { logs: logs2 }  = await factory.createProxy(distribution, version, contractName);
-    const proxyAddress2 = logs2.find(l => l.event === 'ProxyCreated').args.proxy;
-    this.mock2 = await PickACard.new(proxyAddress2);
-
+    this.proxyAddress2 = logs2.find(l => l.event === 'ProxyCreated').args.proxy;
+    this.mock2 = await PickACard.new(this.proxyAddress2);
```

Comment:
what about using `anotherMock` here?

Reply:
I changed `this.proxyAddress2` to `this.anotherProxyAddress` as well, despite the ambiguity as to whether it is a new proxy or a new address of the same proxy, which is in any case also present in `proxyAddress2`.